projects/spring-cloud-commons/build.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+mv $SRC/*.dict $OUT
+
+mkdir -p $JAVA_HOME
+cp -rL "/usr/lib/jvm/java-17-openjdk-amd64/." "$JAVA_HOME" || true
+
+cat > patch.diff <<- EOM
```

Comment:
Can you add a comment here for why this is needed?

Reply:
friendly ping  @onionpsy

Reply:
@oliverchang I opted for a cleaner solution :). Sorry for the delay

Reply:
Did you push that change? I still see the patch in this PR.

I'll merge it for now, but please address this in another PR.